CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
   * [UPGRADE] - Pushgateway has been upgraded from version 1.4.3 to 1.5.1
   * [UPGRADE] - K8s-sidecar used with Grafana has been upgraded from 1.19.5 to 1.22.0
 
+* **Logging**
+  * [FIX] Corrected kubectl commands provided in the logging/bin/change_internal_password.sh to reset Fluent Bit and Elastic Exporter pods.
```

Comment:
Slight tweak to clarify what was changed and correct the component name: 
[FIX] Corrected messages in logging/bin/change_internal_password.sh showing commands to restart Fluent Bit and Elasticsearch Exporter pods.